haystack/components/evaluators/document_map.py

```diff
@@ -4,20 +4,20 @@
 
 
 @component
-class DocumentMeanAveragePrecision:
+class DocumentMeanAveragePrecisionEvaluator:
```

Comment:
I think `DocumentMAPEvaluator` would be better? We already use acronyms, i.e., `LLMEvaluator`, `SASEvaluator`. We can then highlight the full name in the docstring. @julian-risch ?

Reply:
Yes, I like that DocumentMAPEvaluator and DocumentMRREvaluator

Reply:
SASEvaluator could be AnswerSASEvaluator by the way. Not a fan of these long names. Your choice.

Reply:
Works for me. 👍

Reply:
But there's already "answer" in "Semantic answer similarity".